contributing.md

```diff
@@ -81,6 +81,7 @@ If you do so, kindly edit this file; adding the newly used category (**[/contrib
 * Telecommunication
 * ISP
 * Transportation
+* Technology
```

Comment:
Hi @aimamisa, I'm worried about adding Technology as a category because it encompasses a lot of other categories. Can we use just Agritech?

Reply:
Hello @fathermerry, Yea Agritech is fine.